pkg/rpc/server/server.go

```diff
@@ -542,13 +542,17 @@ func (s *Server) getNEP5Transfers(ps request.Params) (interface{}, *response.Err
 	lg := s.chain.GetNEP5TransferLog(u)
 	cache := make(map[util.Uint160]int64)
 	err = lg.ForEach(func(tr *state.NEP5Transfer) error {
+		h, errGetHash := s.chain.GetContractScriptHash(tr.Asset)
+		if errGetHash != nil {
+			return nil
+		}
 		transfer := result.NEP5Transfer{
 			Timestamp: tr.Timestamp,
-			Asset:     tr.Asset,
+			Asset:     h,
 			Index:     tr.Block,
 			TxHash:    tr.Tx,
 		}
-		d, err := s.getDecimals(tr.Asset, cache)
+		d, err := s.getDecimals(h, cache)
```

Comment:
We can impove this to make `cache` store decimals by `ID` (in a separate commit).

Reply:
I thought about that but the case is that even if we use ID for storing decimals, we'll still have to get the contract hash to fill `result.NEP5Transfer.Asset` field. So basically, there's not so mach difference between `ID` and `hash` here. Do we really need to change this?

Reply:
This will allow to cache hashes too. May be a separate issue, though.

Reply:
Somehow I didn't pay attention to hashes caching. Refactored `getDecimals` in a separate commit, now it stores decimals by contract ID and caches contracts hashes also.